mumbleBot.py

```diff
@@ -358,7 +365,8 @@ def is_admin(user):
 
     def get_user_count_in_channel(self):
         own_channel = self.mumble.channels[self.mumble.users.myself['channel_id']]
-        return len(own_channel.get_users())
+        return len(set([user.get_property("name") for user in own_channel.get_users()]).difference(self.bots))
```

Comment:
I suggest not to do so many things in one line. It would be hard for other people to understand your intention.